index.js

```diff
@@ -730,6 +731,8 @@ class Encore {
      * @returns {Encore}
      */
     configureDevServerOptions(callback) {
+        featuresHelper.ensurePackagesExistAndAreCorrectVersion('webpack-dev-server');
```

Comment:
Shouldn't this be done in WebpackConfig or config-generator like the other similar checks ?

Reply:
You're right